MdeModulePkg/Bus/Usb/UsbBusDxe/UsbBus.c

```diff
@@ -4,6 +4,7 @@
 
 Copyright (c) 2004 - 2018, Intel Corporation. All rights reserved.<BR>
 Copyright (C) 2024 Advanced Micro Devices, Inc. All rights reserved.
+Copyright (c) Microsoft Corporation.

```

Comment:
(comment on commit message)

Can we get rid of the passive voice?

"Add new function UsbUpdateDescriptors () which reads the current USB device configuration,
and invoke that as part of UsbIoPortReset ()."

Reply:
Sure